PPAK_CVS/Sources/Scenes/Home/HomeCollectionHeaderView.swift

```diff
@@ -70,38 +69,39 @@ final class HomeCollectionHeaderView: UICollectionReusableView {
     }
 
     pageControl.snp.makeConstraints { make in
-      make.centerY.equalToSuperview().offset(16)
       make.leading.trailing.equalToSuperview().inset(40)
+      make.bottom.equalToSuperview().inset(110)
       make.height.equalTo(65)
     }
 
     searchBar.snp.makeConstraints { make in
       make.bottom.equalToSuperview()
-      make.leading.equalToSuperview().inset(32)
+      make.leading.equalToSuperview().inset(40)
       make.trailing.equalTo(filterButton.snp.leading).offset(-16)
       make.height.equalTo(50)
     }
 
     filterButton.snp.makeConstraints { make in
       make.centerY.equalTo(searchBar)
-      make.trailing.equalToSuperview().inset(32)
+      make.trailing.equalToSuperview().inset(20)
       make.width.height.equalTo(30)
```

Comment:
```suggestion
      make.trailing.equalToSuperview().inset(16)
      make.width.height.equalTo(44)
```
Figma와 맞춰주시면 감사하겠습니다!
이렇게 하시면 버튼의 이미지가 커질텐데, 추후에 Assets에 이미지 파일 모두 1x(24p), 2x(48p), 3x(72p)로 넣을 예정이라 이렇게 넣으면 예쁘게 보입니다 :)

Reply:
![image](https://user-images.githubusercontent.com/97531269/208228229-2ec253b0-a0a9-4199-af24-c0e89736e962.png)

민지님이 피그마에 올려주신 정렬 버튼으로 에셋 1x, 2x, 3x모두 넣었는데 
이렇게 나오더라고요 .. 
이대로 놔둬도 괜찮을까요?

Reply:
@eung7 
아! 네넵 응칠님 브랜치 받아서 확인해봤는데요
ic_sort의 경우, 1x(30p) 2x(60p) 3x(90p)
ic_heartwhite의 경우, 1x(44p) 2x(88p) 3x(132p)
로 수정해서 Asset에 넣고 다시 돌려보니깐 예쁘게 잘 나오더라구요
너무 걱정하지 않으셔도 됩니다ㅎㅎㅎ 제가 추후에 Asset에 이미지 별로 사이즈 맞게 넣어놓을게요~

Reply:
알겠습니다 이 부분은 이대로 놔두도록 할게요!